src/App.js

```diff
@@ -1,16 +1,28 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import { useState } from 'react';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+
+//state variables to use for total red hearts count and to pass to Chatlong and ChatEntry
+  const [redFilledHeartCount, setRedFilledHeartCount] = useState(0);
+
+
   return (
     <div id="App">
       <header>
-        <h1>Application title</h1>
+        <h1>Chat between Vladimir and Estragon</h1>
+        <p>{redFilledHeartCount} ❤️s</p>
       </header>
       <main>
-        {/* Wave 01: Render one ChatEntry component
-        Wave 02: Render ChatLog component */}
+        
+        {/* to render chatlog component */}
+        <ChatLog entries={chatMessages}
+          redFilledHeartCount={redFilledHeartCount}
+          setRedFilledHeartCount={setRedFilledHeartCount}
```

Comment:
It is not necessary to send both the heart count and the function for setting the heart count. Ideally, a function is passed down that will handle the update of the count without needing to also pass down the count. If the count information is being stored in a database, it can get out of sync with what is being sent down to the component. When we pass around the state variable like this, it becomes possible to have different sources of truth.